### DISCUSSION TITLE
是否有办法向模块注入class名

### DISCUSSION THREAD

**Discussion by tsq456** [walinejs/waline#201] [Q&A] [answered]
比如用bootrap的collapse插件默认隐藏评论输入框，只展示评论内容这样的场景。

**Comment by lizheming** [comment, marked as answer]
直接在主 DIV 上增加 class 应该就可以了吧？

**Reply by tsq456** [reply]
是想控制下面的个别类，现在结构是这样的

.v
-- .vpanel
-- .vcount
-- .vpower.txt-right

这时候想单独通过bootrap控制vpanel的隐藏的话 就需要加个类名.collapse

**Reply by lizheming** [reply]
可以自己增加 `#waline.collapse .vpanel { display: none; }` 的样式不走 Bootstrap 的行为处理一下。
